infra/base-images/base-image/Dockerfile

```diff
@@ -29,3 +29,17 @@ ENV WORK=/work
 ENV PATH="$PATH:/out"
 
 RUN mkdir -p $OUT $SRC $WORK && chmod a+rwx $OUT $SRC $WORK
+
+# Steps for ARM cross-compilation
+RUN apt-get install qemu-user wget xz-utils -y && \
```

Comment:
nit: this seems better suited to be in base-clang instead? i.e. keep all the compiler installing steps in the one image.

Can we also uninstall this custom gcc after clang is built (in checkout_build_install_llvm.sh) so we save on both space and avoid potentially using the wrong compiler in project builds?

Reply:
We definitely need qemu in runners+builders because both images need to be able to run targets (some build systems don't work if they cant run binaries they build).
The reason I install the sysroot in both is because there are some things like the linker that are needed to run binaries. Maybe I can get a slimmed down thing that includes the linker but not gcc to include both.